addons/netfox/network-time-synchronizer.gd

```diff
@@ -1,146 +1,244 @@
 extends Node
+class_name _NetworkTimeSynchronizer
 
-## Time between syncs, in seconds.
+## Continuously synchronizes time to the host's remote clock.
 ##
+## Make sure to read the [i]NetworkTimeSynchronizer Guide[/i] to understand the
+## different clocks that the class docs refer to.
+##
+## @tutorial(NetworkTimeSynchronizer Guide): https://foxssake.github.io/netfox/netfox/guides/network-time-synchronizer/
+
+## Time between sync samples, in seconds.
+## [br]
 ## [i]read-only[/i], you can change this in the Netfox project settings
 var sync_interval: float:
```

Comment:
TODO: As suggested by WifeHaver on Discord, we should clamp this value so clients won't overwhelm the server with ping requests, or even malicious players by placing an override.cfg next to the game and setting this value to something overly low.

Reply:
Done.